react-components/src/architecture/concrete/reveal/cad/CadDomainObject.ts

```diff
@@ -52,6 +52,9 @@ export class CadDomainObject extends VisualDomainObject {
 
   protected override removeCore(): void {
     super.removeCore();
-    getRenderTarget(this)?.viewer?.removeModel(this._model);
+    const viewer = getRenderTarget(this)?.viewer;
+    if (viewer?.models.includes(this._model) ?? false) {
```

Comment:
non blocking
Any reason why we use `??`?

Reply:
![image](https://github.com/user-attachments/assets/61c1f676-e29e-40f0-9360-2ac4efd34bbd)